docs/index.md

```diff
@@ -1053,55 +1051,55 @@ Can be specified as a comma-delimited list.
 
 ### `--config <path>`
 
-> *New in v6.0.0.*
+> _New in v6.0.0._
 
 Specify an explicit path to a [configuration file](#configuring-mocha-node-js).
 
 By default, Mocha will search for a config file if `--config` is not specified; use `--no-config` to suppress this behavior.
 
 ### `--opts <path>`
 
-> *Updated in v6.0.0; added `--no-opts`.*
+> _Updated in v6.0.0; added `--no-opts`._
 
 Specify a path to [`mocha.opts`](#mochaopts).
 
 By default, Mocha looks for a `mocha.opts` in `test/mocha.opts`; use `--no-opts` to suppress this behavior.
 
 ### `--package <path>`
 
-> *New in v6.0.0.*
+> _New in v6.0.0._
 
-Specify an explicit path to a [`package.json` file](#configuring-mocha-node-js)  (ostensibly containing configuration in a `mocha` property).
+Specify an explicit path to a [`package.json` file](#configuring-mocha-node-js) (ostensibly containing configuration in a `mocha` property).
```

Comment:
From `mocha --help`:
>  --opts     Path to `mocha.opts`        [string] [default: "./test/mocha.opts"]
>   --package  Path to package.json for config                            [string]

"package.json" should be quoted like the preceding "mocha.opts"

Reply:
please create an issue or fix elsewhere